plugin/helpers.go

```
@@ -69,3 +76,24 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+func (p *HelpersImpl) InstallPluginFromUrl(downloadUrl string, replace bool) (*model.Manifest, error) {
+	parsedUrl, _ := url.Parse(downloadUrl)
```

Please don't ignore the error